db/db_sqlite3/migrations/20191104103306_0.9.0_create_webhooks.sql

```diff
@@ -0,0 +1,14 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE IF NOT EXISTS "webhooks" (
+    "id" integer primary key autoincrement,
+    "title" varchar(255),
+    "url" varchar(1000),
+    "secret" varchar(255)
```

Comment:
You're missing the `is_active` column in this migration.

Reply:
I'm not, you're looking at the old code.

Reply:
Also, why are you using an integer field for `is_active`? We use boolean fields in migrations now and this seems like another good use case for that.

Reply:
is_active is boolean now

Reply:
I see. There're 2 migrations, in one of them it's missing

Reply:
This is the latest code right? Like, the code right here that we're commenting on is for the SQLite migrations, and it doesn't have `is_active`. MySQL does, but SQLite doesn't.

Reply:
```
SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true).
```

https://www.sqlite.org/datatype3.html



Sqlite doesn't support boolean, why are we using them in all the migration for Sqlite?

Reply:
It doesn't have a separate concept for booleans, but does support a `boolean` data type- it just represents them internally as integers:

```
sqlite> .schema webhooks
CREATE TABLE IF NOT EXISTS "webhooks" (
    "id" integer primary key autoincrement,
    "title" varchar(255),
    "url" varchar(1000),
    "secret" varchar(255),
    "is_active" boolean
);
sqlite> select * from webhooks;
1|Test Webhook|http://127.0.0.1:9999/webhook|mysecret|0
```

I use them both for clearer database structure, since it's clear that this is a true/false kind of field rather than an integer, as well as to try and avoid any possible weird issues with the underlying ORM, gorm.

Reply:
Fixed